languages/ruby/exercises/concept/arrays/.docs/introduction.md

```diff
@@ -0,0 +1,104 @@
+In Ruby, **arrays** are ordered, integer-indexed collections of any object. Array indexing starts at 0, as in C or Java. A negative index is assumed to be relative to the end of the array—that is, an index of -1 indicates the last element of the array, -2 is the next to last element in the array, and so on.
+Ruby arrays mixed in the [Enumerable module][enumerable-module] which provides collection classes with several traversal and searching methods, and with the ability to sort.
+
+### Create array.
+
+- An array in Ruby can contain different types of objects.
+
+```ruby
+array = [1, "two", 3.0] #=> [1, "two", 3.0]
+```
+
+- An array can also be created by explicitly calling ::new
+
+```ruby
+array = Array.new #=> []
+Array.new(3)      #=> [nil, nil, nil]
+```
+
+### Element Assignment
+
+- Sets the element at index, or replaces a subarray from the start index for length elements.
+
+```ruby
+a = Array.new
```

Comment:
```suggestion
a = ["", "", "", "", ""]
```

Reply:
Let's not add the Array.new syntax for now, as we haven't taught `nil` anywhere, and we probably don't want to add that as a prereq just for this example.